src/main/java/frc/robot/RobotMap.java

```diff
@@ -0,0 +1,12 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+public class RobotMap {
+    public static final class Pneumatics {
+        public static final int PCM = 41;
+        public static final PneumaticsModuleType PCM_TYPE = PneumaticsModuleType.CTREPCM;
```

Comment:
Are you sure that this is the PCM type we are planning on using this year?

Reply:
Yes I'm sure that's it the correct PCM